rust/processor/src/db/common/models/default_models/write_set_changes.rs

```diff
@@ -19,6 +19,7 @@ use aptos_protos::transaction::v1::{
 };
 use field_count::FieldCount;
 use serde::{Deserialize, Serialize};
+use tracing::log::error;
```

Comment:
nit: tracing re-exports the macros at the top level
```suggestion
use tracing::error;
```